ONSdigital/design-system
# Unsupported browser banner

### What is the context of this PR?
Resolves #2296 

### How to review
- Check the documentation
- Check tests pass locally
- Check banner appears on IE8-11 only
